src/query/trigger.cpp

```diff
@@ -158,13 +158,13 @@ Trigger::Trigger(std::string name, const std::string &query, const storage::Prop
       event_type_{event_type},
       owner_{std::move(owner)} {
   // We check immediately if the query is valid by trying to create a plan.
-  GetPlan(db_accessor);
+  GetPlan(db_accessor, 0);
```

Comment:
`DatabaseAccessProtector` is `std::any` which is why this compiles...but it doesn't do any protection

Reply:
removed the db_acc so this argument is also removed.